var/spack/repos/builtin/packages/zfp/package.py

```diff
@@ -62,55 +73,86 @@ class Zfp(CMakePackage, CudaPackage):
         "use smaller for finer rate granularity. "
         "Use 8 for H5Z-ZFP filter.",
     )
+
     variant(
-        "strided", default=False, description="Enable strided access for progressive zfp streams"
+        "daz",
+        default=False,
+        description="Denormals are zero: "
+        "Treat denormal-only blocks as containing "
+        "all zeroes",
     )
-    variant("aligned", default=False, description="Enable aligned memory allocation")
-    variant("twoway", default=False, description="Use two-way skew-associative cache")
+
     variant(
         "fasthash",
         default=False,
         description="Use a faster but more collision prone hash function",
     )
+
     variant("profile", default=False, description="Count cache misses")
 
+    variant(
+        "round",
+        default="never",
+        values=("never", "first", "last"),
+        multi=False,
+        when="@1.0.0:",
+        description="EXPERIMENTAL: Set coefficient rounding method",
+    )
+
+    variant(
+        "strided", default=False, description="Enable strided access for progressive zfp streams"
+    )
+
+    variant(
+        "tight-error",
+        default=False,
+        when="@1.0.0:",
+        description="EXPERIMENTAL: Use tighter error bound when rounding is enabled",
+    )
+
+    variant("twoway", default=False, description="Use two-way skew-associative cache")
+
     # Conflicts
```

Comment:
I would suggest we handle these "conflicts" via `when` in the variants, just like we're doing with the new `round` variant, etc.  The main conflict appears to be combinations of variants, like with `round` and `tight-error`.